--- a/CHANGES
+++ b/CHANGES
@@ -1,10 +1,21 @@
+4.17 2005.09.27
+---------------
+* removed eurosans package I did never get it to actually display a symbol with all the
+  other euro packages loaded at the same time.
+* made mentions of Latin and Cyrillic consistently capitalized
+* added note on Latin modern outline fonts to replace EC in pdfs
+* added proc and minimal class
+* mention companion 2nd edition
+* update for ifpdf section from Morten Høgholm
+* print from 4.1 to chapter 5 for pure latex graphics
+
 4.16 2005.05.08
-----
-* include propperly renderd PDF version with the distribution
+---------------
+* include properly rendered PDF version with the distribution
 
 4.15 2005.05.08
 ----
-* replced pdfscreen with beamer for presentations. Used text from the french
+* replaced pdfscreen with beamer for presentations. Used text from the french
   lshort by Daniel Flipo as a basis for this.
 * eqref requires amslatex 
 * use \centering to  put figure into the center as begin/end{center} adds
