--- a/CHANGES
+++ b/CHANGES
@@ -1,3 +1,37 @@
+4.23
+-----
+Math Chapter reworked by Lan Thuy Pham:
+
+* The amsmath package is now introduced in the beginning
+  of the chapter, and it is assumed that the reader has loaded the package.
+
+* Wrote a short intro to AMS-LaTeX.
+
+* Added a bit about: partial, (wide)hat, bar, pieceswise functions, and from amsmath:
+  dfrac, tfrac, tag, different matrix environments.
+
+* Modified examples: equation, equation*, \text, Greek letters, exponents
+  ans subscripts, \(c/l)dot(s), under/overline, under/overbrace, \vec,
+  \frac, \binom, stackrel, delimiters, align/eqnarray, long equations,
+  array, matrix, spacing, phantoms, (bold) math fonts, \displaystyle...
+
+* Removed: \( \) and the "math" environment, displaymath, subarray,
+  eqnarray, some of the array and phantom examples, textrm.
+
+* Added Note about upper case Greek letters.
+
+* Added a new table: Arrows as Accents.
+
+* Rearranged the "Delimiters"-table a bit.
+
+* Added \clearpage after the non-amsmath stuff (to stop LaTeX from complaining).
+
+* Added a couple of labels.
+
+*  Added a reference to Math Fonts (\ref{mathfonts}) at Math Alphabets.
+
+* Rearranged some of the ams-tables to optimise the space.
+
 4.22 2007.06.30
 ---------------
 * fixed my email address
