--- a/CHANGES
+++ b/CHANGES
@@ -1,8 +1,27 @@
-3.22
-----
-From Tobi:
-* fix for messed up layout in prev issue (flushleft in contrib and font index)
-
+4.0
+---
+* many minor fixes and edits in pagebreaks
+* added note on cm-super
+* punctuation, wording, big patch from Baron Schwartz <bps7j@cs.virginia.edu>
+* Translated and integreated the pdfTeX description from the 
+  franch version of lshort.
+* Added descrion of the .fd files
+* \part is available in article too
+* small backports from the french translation
+* drop atop and choose. Replace by binom
+* put notes on Float Placing Permissions where they belong
+* added eqref
+* mention landscape option of default document classes
+* mathcal update
+* changed section headings in first sample
+* drop latex structure figure from the intro as it is way to complex for a beginner to understand.
+* drop LaTeX2e story ... who care these days
+* added note on swiss use of >>guillemets<<
+* added note on eurosymb
+* added note on dvipdf
+* added article on Korean Support
+* added Xy Pic intro
+* more symbols aded -- alex mai
 3.21
 ----
 From Tobi
