--- a/Makefile
+++ b/Makefile
@@ -47,10 +47,5 @@ dist:
 	(gecho "Robin,\n\nI have uploaded lshort-$(VERS) to ftp.tex.ac.uk:/incoming/lshort-$(VERS).\n\nAn Announcement is not necessary\nCheers tobi\n\n\n--";fortune -s shakes goethe) | mailx -s "Lshort Upload (note the quote)" ctan@dante.de
 	(gecho "Folks,\n\nI have created lshort-$(VERS). It is available from http://people.ee.ethz.ch/~oetiker/lshort.\n\nCheers tobi\n\n\n--";fortune -s shakes goethe) | mailx -s "Lshort $(VERS)" `cat TRLIST`
 
-quick: $(FILES)
-         (TEXINPUTS=`pwd`/src:$(TEXINPUTS); export TEXINPUTS; cd texbuild;\
-         latex lshort; mv lshort.dvi ..)
-
 clean:
 	rm -rf texbuild pdfbuild
-
