--- a/Makefile
+++ b/Makefile
@@ -18,12 +18,12 @@ all: lshort.dvi lshort.ps lshort-book.ps lshort.pdf
 
 lshort.dvi: $(FILES)
 	-mkdir texbuild
-	(TEXINPUTS=.:`pwd`/src:${TEXINPUTS:-:}; export TEXINPUTS; cd texbuild; \
-	$(LATEX) lshort; $(LATEX) lshort; $(MAKEINDEX) -s ../src/lshort.ist lshort; \
-	$(LATEX) lshort; $(LATEX) lshort; mv lshort.dvi ..)
+	(TEXINPUTS=.:`pwd`/src:${TEXINPUTS:-:} && export TEXINPUTS && cd texbuild && \
+	$(LATEX) lshort && $(LATEX) lshort && $(MAKEINDEX) -s ../src/lshort.ist lshort; \
+	$(LATEX) lshort && $(LATEX) lshort && mv lshort.dvi ..)
 
 lshort.ps: lshort.dvi
-	dvips -Pcmz -olshort.ps lshort.dvi
+	(T1FONTS=.:`pwd`/eurofont: && export T1FONTS && dvips -Pcmz -olshort.ps lshort.dvi )
 	rm texbuild/*
 
 lshort-book.ps: lshort.ps
