--- a/Makefile
+++ b/Makefile
@@ -51,7 +51,7 @@ lshort-a5book.pdf: lshort-a5book.ps
 
 lshort.pdf: $(FILES)
 	-mkdir pdfbuild
-	(T1FONTS=.:`pwd`/eurofont: && export T1FONTS && TEXINPUTS=.:`pwd`/src:${TEXINPUTS:-:}&&export TEXINPUTS&& cd pdfbuild&& \
+	(T1FONTS=.:`pwd`/eurofont: && export T1FONTS && TEXINPUTS=.:`pwd`/src:`pwd`/euro:${TEXINPUTS:-:}&&export TEXINPUTS&& cd pdfbuild&& \
 	$(PDFLATEX) lshort&& $(PDFLATEX) lshort&& \
 	$(MAKEINDEX) -s ../src/lshort.ist lshort&&$(PDFLATEX) lshort&& \
 	(thumbpdf --resolution 10 lshort.pdf && $(PDFLATEX) lshort)&& \
