--- a/Makefile
+++ b/Makefile
@@ -45,8 +45,8 @@ src/title.tex: Makefile
 	perl fixdate.pl $(VERS) < src/title.tex > src/title.tex2 && mv src/title.tex2 src/title.tex
 
 quick: $(FILES)
-	(TEXINPUTS=`pwd`/src:$(TEXINPUTS); export TEXINPUTS; cd texbuild; \
-      $(LATEX) lshort; mv lshort.dvi ..)
+	(TEXINPUTS=`pwd`/src:$(TEXINPUTS)&& export TEXINPUTS&& cd texbuild&& \
+        $(LATEX) lshort&& mv lshort.dvi ..)
 
 
 tar:	src/title.tex
