--- a/Makefile
+++ b/Makefile
@@ -32,6 +32,12 @@ lshort-book.ps: lshort.ps
 			out.ps lshort-book.ps
 	rm out.ps
 
+lshort-a5book.ps: lshort-a5.ps
+	( psbook lshort-a5.ps | psnup -pa4 -s1 -2 | pstops "2:0,1U(21cm,29.7cm)" >lshort-a5book.ps )
+
+lshort-a5book.pdf: lshort-a5book.ps
+	 ps2pdf14 -sPAPERSIZE=a4 lshort-a5book.ps lshort-a5book.pdf
+
 lshort.pdf: $(FILES)
 	-mkdir pdfbuild
 	(T1FONTS=.:`pwd`/eurofont: && export T1FONTS && TEXINPUTS=.:`pwd`/src:${TEXINPUTS:-:}&&export TEXINPUTS&& cd pdfbuild&& \
