--- a/Makefile
+++ b/Makefile
@@ -19,22 +19,22 @@ all: lshort.ps lshort-book.ps lshort.pdf lshort-a5book.pdf
 
 lshort.dvi: $(FILES)
 	-mkdir texbuild
-	(TEXINPUTS=.:`pwd`/src:${TEXINPUTS:-:} && export TEXINPUTS && cd texbuild && \
+	(TEXINPUTS=.:`pwd`/src:`pwd`/euro:${TEXINPUTS:-:} && export TEXINPUTS && cd texbuild && \
 	$(LATEX) lshort && $(LATEX) lshort && $(MAKEINDEX) -s ../src/lshort.ist lshort; \
 	$(LATEX) lshort && $(LATEX) lshort && mv lshort.dvi ..)
 
 lshort-a5.dvi: $(FILES)
 	-mkdir texbuild
-	(TEXINPUTS=.:`pwd`/src:${TEXINPUTS:-:} && export TEXINPUTS && cd texbuild && \
+	(TEXINPUTS=.:`pwd`/src:`pwd`/euro:${TEXINPUTS:-:} && export TEXINPUTS && cd texbuild && \
 	$(LATEX) lshort-a5 && $(LATEX) lshort-a5 && $(MAKEINDEX) -s ../src/lshort.ist lshort-a5; \
 	$(LATEX) lshort-a5 && $(LATEX) lshort-a5 && mv lshort-a5.dvi ..)
 
 lshort.ps: lshort.dvi
-	(T1FONTS=.:`pwd`/eurofont: && TEXFONTMAPS=.:`pwd`/eurofont: && export T1FONTS TEXFONTMAPS && $(DVIPS) -j -Pcmz -olshort.ps lshort.dvi )
+	(T1FONTS=.: && TEXFONTMAPS=.: && export T1FONTS TEXFONTMAPS && $(DVIPS) -j -Pcmz -olshort.ps lshort.dvi )
 	rm texbuild/*
 
 lshort-a5.ps: lshort-a5.dvi
-	(T1FONTS=.:`pwd`/eurofont: && TEXFONTMAPS=.:`pwd`/eurofont: && export T1FONTS TEXFONTMAPS  && $(DVIPS) -j -Pcmz -ta5 -olshort-a5.ps lshort-a5.dvi )
+	(T1FONTS=.: && TEXFONTMAPS=.: && export T1FONTS TEXFONTMAPS  && $(DVIPS) -j -Pcmz -ta5 -olshort-a5.ps lshort-a5.dvi )
 	rm texbuild/*
 
 lshort-book.ps: lshort.ps
