--- a/Makefile
+++ b/Makefile
@@ -72,6 +72,8 @@ tar:	$(FILES)
 	rm lshort-$(VERS)
 
 dist:	all tar
+	echo press enter to rsync
+	read x
 	rsync  lshort-$(VERS).src.tar.gz CHANGES README lshort.pdf lshort-a5book.pdf james:public_html/latex/
 	lftp -e 'cd incoming;mkdir lshort-$(VERS);cd lshort-$(VERS);mput lshort-$(VERS).src.tar.gz CHANGES README lshort-book.ps lshort.dvi lshort.pdf lshort.ps;quit' ftp.tex.ac.uk
 	(echo -e "Robin,\n\nI have uploaded lshort-$(VERS) to ftp.tex.ac.uk:/incoming/lshort-$(VERS).\n\nIf you think it is appropriate, announce it please.\n\nThanks and cheers\ntobi\n\n\n--";fortune -s shakes goethe) | mailx -s "Lshort Upload (note the quote)" ctan@dante.de
