--- a/README
+++ b/README
@@ -31,6 +31,10 @@
        will format without problem. After latexing the document twice, 
        you should use makeindex to generate a new index file.
 
+       (to get the adobe euro symbol right you need to install the 
+        ftp://ftp.adobe.com/pub/adobe/type/win/all/eurofont.exe as described in
+        http://www.ctan.org/tex-archive/fonts/euro/latex/europs/README.europs)
+
     The preformatted versions of the document are set for double sided
     printing on A4 paper. If you want a single sided version, you need to
     replace "twoside" by "oneside" in the parameters list of the
