--- /dev/null
+++ b/REMARKS-mpg
@@ -0,0 +1,80 @@
+# Style file (implementation questions)
+
+Why use url.sty and not just hyperref? Is it a matter of taste or does it really
+add features?
+
+Looks like the \@tildeORI things is now useless, is it?
+
+Why save \pageref at the beginning?
+
+mylayout.sty ? fancy.sty ?
+
+# Style file (features)
+
+Using mirror.ctan.org to make clickable CTAN: urls (see \CTAN)
+
+Using ctan.org/pkg/PKGNAME as link for package names (see \package and \pai).
+
+# Style file (implementation)
+
+Use fancyvrb + active chars for lscommand and code environments.
+
+New command \changelabels for descriptions.
+
+New \tti and \tth commands for things in tt in index.
+
+Imported \iei and \fei from typeset.tex
+
+# things.tex
+
+202: add 'this file is refered to as the "source file", or simply "source"'?
+
+250: why not advertise \textbackslash by textcomp? Shifting to math mode for
+anything else but math is a bad habit (\everymath, \mathsurround, etc).
+
+450 & Co: why not simply consider pdflatex as the normal compilation mode.
+Everyone knows how to view a pdf... And anyway nearly everyone will consider
+only the pdf as the final result (even pstricks addicts generally agree that
+dvi/ps are just intermediate steps).
+
+454: AFAIK, yap only works with MikTeX but you can also have TeX Live on windows
+(which unfortunately doesn't come with a dvi viewer).
+
+464: Actually dvipdf comes with GhostScript. Only (x)dvipdfm(x) come with the
+TeX distro. Anyway, this is another point that make me think pdflatex should be
+introduced instead: all those new commands and file formats look so scary to a
+newcomer.
+
+511: Why not advertise beamer?
+
+602: texdoc is specific to teTeX and TeXLive (hence MacTeX), not really to Unix
+(TeXLive exsists on windows). And MikTeX provides a similar thing named mthelp,
+worth mentionning.
+
+609 etc: every decent distro comes with the doc compiled, and the CTAN holds a
+pdf version of every packages's doc (or is trying to). So I would just mention
+doc.pdf etc, remember texdoc/mthelp, and say nothing about compiling dtx.
+
+721: here too, mention pdf as well as dvi. 
+
+# typeset.tex
+
+181: may be worth mentionning that the words in the list should be separated by
+spaces (one could otherwise be tempted to try comas).
+
+310: one could also mention siunitx in this context
+
+330: I think the \emph around official is a typo, should be \verb
+
+326 etc: since the 2nd possibility is commented out, probably it would be better to say one more choice?
+
+397: are you sure that an \mbox is necessary? I think {} is enough.
+
+516 etc: I disagree on tying encodings to platforms. Nowadays, most text editors (at least on Unix and Mac) allow you to choose your encoding. I rephrased the whole paragraph.
+
+523: here too, mention pdf rather than dtx. Also mention utf8ienc.pdf.
+
+530 table: drop dos+os/2, and after tehtable add a remark about
+latin1-latin9-cp1252 (esp. usefull for French) and mention utf-8.
+
+542: I think ucs is obsolete. inputenc handles utf8 correctly anyway.
