--- a/TRANSLATIONS
+++ b/TRANSLATIONS
@@ -10,7 +10,7 @@ French          Matthieu Herrb <matthieu@laas.fr>
 
 French II       Daniel FLIPO <Daniel.Flipo@univ-lille1.fr> 
             (Much more up to date)
-        --> http://www.univ-lille1.fr/labo-stat-proba/df/doc/dfshort.pdf
+        --> http://daniel.flipo.free.fr/doc/dfshort/
 
 Spanish         Tom'as Bautista <bautista@cma.ulpgc.es>
         --> CTAN:info/lshort/spanish
