--- a/TRANSLATIONS
+++ b/TRANSLATIONS
@@ -46,6 +46,9 @@ Czech           Michal Ko\v{c}er and Pavel S\'ykora (kocer@ipex.cz)
 Bulgarian       Stefka Romanova Karakoleva <skarakoleva@ru.acad.bg>
         --> CTAN:info/lshort/bulgarian
 
+Romanian        George Pauliuc <gpauliuc@yahoo.com>
+                 (Stated his intention todo one)
+
 Slovak          Jan Busa <busaj@ccsun.tuke.sk>
         --> not published
 
