--- a/TRANSLATIONS
+++ b/TRANSLATIONS
@@ -22,11 +22,14 @@ Polish		Stanis/law Wawrykiewicz <staw@eti.pg.gda.pl>
 		Janusz Go{\l}dasz, Ryszard Kubiak and Tomasz Przechlewski
 	--> CTAN:info/lshort/polish
 
-Portugese       Demerson Andre Polli <polli@linux.ime.usp.br>
+Brasilian- Portugese  Demerson Andre Polli <polli@linux.ime.usp.br>
         --> http://www.linux.ime.usp.br/~polli
 
+Portugese       Alberto Manuel Brandao Simoes <albie@alfarrabio.di.uminho.pt>
+        --> http://alfarrabio.di.uminho.pt/~albie/lshort
+
 Korean		Karnes Kim <karnes@doeun.pe.kr>
-	--> http://www.doeun.pe.kr/frames/lshort.html
+	--> CTAN:info/lshort/korean
 
 Italian         Gilda Guild <guild-project@bigfoot.com>
         --> CTAN:info/lshort/italian
