--- a/TRANSLATIONS
+++ b/TRANSLATIONS
@@ -53,7 +53,10 @@ Japanese        NOMURA Masataka <nomura@cc.kshosen.ac.jp>
 Korean           Karnes Kim karnes@doeun.pe.kr
         --> http://www.doeun.pe.kr/frames/lshort.html
         --> CTAN:/tex-archive/info/lshort/korean/
->
+
+Swedisch         Anders Källström <andersk@math.uu.se>
+                 (Stated his intention todo one)
+
 
 If you know of other translations, please tell me. 
 
