--- a/TRANSLATIONS
+++ b/TRANSLATIONS
@@ -18,6 +18,9 @@ Spanish         Tom'as Bautista <bautista@cma.ulpgc.es>
 Russian         Boris Tobotras <tobotras@jet.msk.su>
         --> CTAN:info/lshort/russian
 
+Ukranian        Maksym Polyakov <polyama@myrealbox.com>
+        --> http://www.auburn.edu/~polayma/lshort
+
 Polish		Stanis/law Wawrykiewicz <staw@eti.pg.gda.pl>
 		Janusz Go{\l}dasz, Ryszard Kubiak and Tomasz Przechlewski
 	--> CTAN:info/lshort/polish
